src/utils/AddressUtils.ts

```diff
@@ -223,8 +223,20 @@ export class EvmAddress extends Address {
   }
 
   // Constructs a new EvmAddress type.
-  static from(hexString: string): EvmAddress {
-    return new this(utils.arrayify(hexString));
+  static from(address: string, encoding: "base16" | "base58" = "base16"): EvmAddress {
```

Comment:
nb. needs test; will follow up with that.

Reply:
https://github.com/across-protocol/sdk/pull/974/commits/b8aedadc4cd32767d11c2cb3a6870433cd4b3e45